console/forms.py

```diff
@@ -27,8 +28,18 @@ def validate_password(form, field):
 
 
 class StoreForm(Form):
-    tx_id = StringField('tx_id', validators=[
-        Length(max=36),
-        Regexp('^[0-9a-f\-]{36}$|^$', message="Field can only contain hexadecimal characters (0-9a-f) and dash (-)"),
-        Regexp('[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}|^$', message="Field must be in the form {8}-{4}-{4}-{4}-{12}"),
-    ])
+    tx_id = StringField('tx_id')
+    ru_ref = StringField('ru_ref')
+    survey_id = StringField('survey_id')
```

Comment:
Would it be worth trying to add these for Datetime as well?

Reply:
Looking at it now, should be possible but its easier said then done!

Reply:
I've created https://github.com/ONSdigital/sdx-console/tree/retain-store-input-values .  I've got it mostly working, but when it tries to render the page again after the search it falls over (I suspect it worked all this time because we weren't repopulating the field again after the search...).  Can I get this merged in and then work on this separately?